BloomFilter.md

```diff
@@ -132,10 +132,11 @@ to default row group size. It is also recommended to expose the ability for sett
 parameters to end users.
 
 #### File Format
-The Bloom filter data of a column chunk, which contains the size of the filter in bytes, the
-algorithm, the hash function and the Bloom filter bitset, is stored near the footer. The Bloom
-filter data offset is stored in column chunk metadata. Here are Bloom filter definitions in
-thrift:
+Each multi-block Bloom filter is required to work for only one column chunk. The data of a multi-block Bloom
+filter contains a header of Bloom filter, which must includes the size of the filter in bytes, the algorithm,
```

Comment:
when you say "a header of Bloom filter", do you mean "the header of one Bloom filter"?

Also, "include the size", not "includes the size".